pages/common/cups.md

```diff
@@ -0,0 +1,25 @@
+# CUPS
+
+> Open-source printing system.
+> CUPS isn't a command, but a set of commands. See the pages below.
+> More information: <https://www.cups.org/index.html>.
+
+- Show the tldr page for running the CUPS daemon:
```

Comment:
Not sure why you need to enclose CUPS inside a command here, since CUPS is the project name too (suggestion for minor changes like this it is better to make a comment review than requesting changes). Will fix it in my review covering all examples.